src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
+    fun withFailedToStart(value: Boolean): BuildLocator
```

Comment:
Calls like `withFailedToStart(false)` will look confusing. And why do you need this method at all? If `value` is `false` it's just a no-op, and if `value` is `true` it's equivalent to the previous method.